config/settings.yml

```diff
@@ -87,7 +87,7 @@ aca:
   market_kinds: ["shop"]
   nationwide_markets: false
   general_agency_enabled: false
-  dental_plan_years_available: ["2018"]
+  dental_plan_years_available: [2018]
```

Comment:
Leave this in a string. There are other places in the application where we are comparing with a string value.